assets/layers/scouting_groups/scouting_groups.json

```diff
@@ -0,0 +1,291 @@
+{
+  "credits": "Osmwithspace",
+  "filter": [
+    {
+      "id": "association",
+      "options": [
+        {
+          "question": {
+            "en": "scout association",
+            "de": "Bund/Verband"
+          }
+        },
+        {
+          "question": {
+            "en": "Bund der Pfadfinderinnen und Pfadfinder"
```

Comment:
This "english" translation looks very German to me...

Reply:
It's basically a proper name. Should I just assign a German tag instead?